app/src/main/java/com/example/easysurvey/MainActivity.kt

```diff
@@ -23,40 +28,78 @@ class MainActivity : ComponentActivity() {
             EasySurveyTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background) {
-                    HomepageButtons()
+                    homepageButtons()
                 }
             }
         }
     }
 }
 
 @Composable
-fun HomepageButtons() {
+fun homepageButtons() {
     val text = "You clicked this button"
     val duration = Toast.LENGTH_LONG
     val context = LocalContext.current
-    Column{
-        Button(
-            onClick = { 
-                Toast.makeText(context, text, duration).show()
+    Column(
+        Modifier.fillMaxHeight(),
+        verticalArrangement = Arrangement.SpaceBetween)
+    {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(130.dp)
+                .padding(vertical = 30.dp, horizontal = 30.dp),
+            horizontalArrangement = Arrangement.Center,
+        ) {
+            Column(horizontalAlignment = Alignment.CenterHorizontally) {
+                Text(
+                    text = stringResource(id = R.string.app_name),
+                    style = TextStyle(fontSize = 40.sp)
+                )
+                Text(
+                    text = stringResource(id = R.string.app_description),
+                    style = TextStyle(fontSize = 13.sp)
+                )
             }
-        ){ Text(text = "Take a Survey")}
-        Button(
-            onClick = {
+        }
```

Comment:
This column holds the title on the splash screen. The title includes the name of the application along with a brief description/slogan.